server/src/main/java/org/candlepin/controller/refresher/builders/NodeFactory.java

```diff
@@ -67,23 +71,45 @@ public NodeFactory addBuilder(NodeBuilder builder) {
     }
 
     /**
-     * Sets the mapper to use for mapping nodes created by this factory.
+     * Adds a mapper to this factory. The mapper will be used to provide entities during node
+     * construction for the class returned by the mapper's <tt>getEntityClass</tt> method.
      *
      * @param mapper
-     *  the mapper to use for mapping nodes
+     *  the mapper to add to this factory
      *
      * @throws IllegalArgumentException
      *  if the provided mapper is null
      *
      * @return
      *  a reference to this node factory
      */
-    public NodeFactory setNodeMapper(NodeMapper mapper) {
+    public NodeFactory addMapper(EntityMapper mapper) {
```

Comment:
Could we do something about our use of raw types? It's starting to feel a bit excessive. We could at least use the wildcards.

Reply:
I've added wildcards where appropriate, but this doesn't really buy us much outside of one weird compiler error that required a random cast to the same type.